governance/default_test.go

```diff
@@ -57,6 +62,7 @@ var tstData = []voteValue{
 	{k: "istanbul.policy", v: uint64(2), e: false},
 	{k: "istanbul.policy", v: float64(1.2), e: false},
 	{k: "istanbul.policy", v: float64(1.0), e: false},
+	{k: "istanbul.policy", v: true, e: false},
```

Comment:
@mckim19 
Just curious though,
is there any test case to check expected result is `true` about `istanbul.policy`? 

It would be great if there a case for expecting `true` as result.

Reply:
Except for this, everything looks great for me :)

Reply:
@aeharvlee `istanbul.policy` is registered as forbidden key. I think this is the reason that voting at `istanbul.policy` always returns false.

https://github.com/klaytn/klaytn/blob/ee71e56f0548eba487ed218f581ffd3cf3a23832/governance/default.go#L70-L74

Reply:
@yoomee1313 
Now it's clear. Thanks yoomee.

cc. @mckim19